src/Reducers/initialState.js

```diff
@@ -33,7 +33,7 @@ const initialState = {
       'Domain',
       'Owner',
       'Contains',
-      'Doesn\'t contain',
+      "Doesn't contain",
```

Comment:
Was this broken before by using single quotes with the escape?

Reply:
I don't think I  changed  this it might have been prettier

Reply:
yea just tested its a linter auto formatting